chain/epoch-manager/src/adapter.rs

```diff
@@ -376,6 +376,15 @@ pub trait EpochManagerAdapter: Send + Sync {
     ) -> Result<bool, EpochError>;
 
     fn will_shard_layout_change(&self, parent_hash: &CryptoHash) -> Result<bool, EpochError>;
+
+    /// Returns a vector of all hashes in the epoch ending with `last_block_info`.
```

Comment:
What are epoch hashes? Block hash of every block? Block hash of every block that EpochSync uses?

Edit: Looked at the implementation. Ok, it really is the hash of every block in an epoch.